src/cadence/mainnet/testGetNFTs.sh

```diff
@@ -216,6 +216,11 @@ BarterYard)
   flow scripts execute getNFTs.cdc --args-json '[{ "type": "Address", "value": "0xa95b021cf8a30d80" }, { "type": "Dictionary", "value": [{ "key": { "type": "String", "value": "BarterYardPack" }, "value": { "type": "Array", "value": [{ "type": "UInt64", "value": "365" }] } }] }]' --network mainnet
   ;;
 
+BarterYardClubWerewolf)
```

Comment:
Seeing the error below on running the test:
❌ Invalid argument: failed to execute script: failed to execute script at block (53d71d0ceb6af480030ca88fcc222a0a5cc8ad4661bec81cec360abd9cd82325): [Error Code: 1101] cadence runtime error Execution failed:
error: pre-condition failed: NFT not in collection
   --> 28abb9f291cadaf2.BarterYardClubWerewolf:453:26

Reply:
fixed ;-)